charts/k8s-watcher/values.yaml

```diff
@@ -29,9 +29,6 @@ proxy:
   no_proxy: ""
 
 resources:
-  limits:
-    cpu: 1
-    memory: 4Gi
```

Comment:
Instead of removing the field completely, change to `limits: {}`
This way it will allow overriding a specific subfield `--set resources.limits.cpu=1` without getting and error tha `limits` doesnt exists